src/txmempool.cpp

```diff
@@ -619,13 +614,12 @@ static void CheckInputsAndUpdateCoins(const CTransaction& tx, CCoinsViewCache& m
 
 void CTxMemPool::check(const CCoinsViewCache *pcoins) const
 {
-    LOCK(cs);
-    if (nCheckFrequency == 0)
-        return;
+    if (m_check_ratio == 0) return;
```

Comment:
You can join the `if (m_check_ratio == 0)` with the `return;` line in the first commit instead of the second commit.

Reply:
updated 👍